### PR TITLE
API-6705 - Bump to scala 2.13

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: app/uk/gov/hmrc/apiplatformjobs/models/UnusedApplication.scala [hunk 1]
```diff
@@ -52,18 +50,28 @@ case class UnusedApplication(
     scheduledDeletionDate: LocalDate
   )
 
-object Environment extends Enumeration {
-  type Environment = Value
-  val SANDBOX, PRODUCTION = Value
+import enumeratum.{Enum, EnumEntry, PlayJsonEnum}
```
Comment (johnsgp) [inline app/uk/gov/hmrc/apiplatformjobs/models/UnusedApplication.scala]: Could we do this without enumeratum?

File: test/uk/gov/hmrc/apiplatformjobs/repository/UnusedApplicationsRepositorySpec.scala [hunk 2]
```diff
@@ -125,6 +125,7 @@ class UnusedApplicationsRepositorySpec
           )
           .toFuture()
       )
+      println("XXXX " + await(unusedApplicationRepository.collection.countDocuments().toFuture()))
```
Comment (johnsgp) [inline test/uk/gov/hmrc/apiplatformjobs/repository/UnusedApplicationsRepositorySpec.scala]: I guess this was for debugging only?

Comment (AndySpaven) [reply test/uk/gov/hmrc/apiplatformjobs/repository/UnusedApplicationsRepositorySpec.scala]: No - it's what makes the whole thing work :-)

Comment (AndySpaven) [reply test/uk/gov/hmrc/apiplatformjobs/repository/UnusedApplicationsRepositorySpec.scala]: As are the other two printlns... :-O

File: build.sbt
Comment (johnsgp) [inline build.sbt:1]: This file needs something to ignore the "unused imports" in app.routes etc.

**Review by johnsgp — APPROVED** [review]

**Pull request merged**